renderer/src/VulkanRenderer.hpp

```diff
@@ -119,13 +119,12 @@ class R_PUBLIC_API VulkanRenderer {
     bool createCommandPool(VkDevice logical_device);
     bool createCommandBuffers(VkDevice logical_device);
     bool createSyncObjects(VkDevice logical_device);
-    bool recreateSwapchain(VkPhysicalDevice device, VkDevice logical_device, bool handle_rendering_objects = false);
+    bool recreateSwapchain(VkPhysicalDevice device, VkDevice logical_device, bool objects_added = false, bool resized = false);
```

Comment:
Consider making optional arguments, obligatory, to avoid strange unexpected behaviors.